addons/sale_stock/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,45 @@
+---Fields in module 'sale_stock'---
+sale_stock   / procurement.group        / sale_id (many2one)            : NEW relation: sale.order
+# Nothing to do
```

Comment:
You need to fill this field, or everything previous to the migration will not be connected, including pending sales orders. Get it from the inverse `procurement_group_id` field in sale.order or old procurement order.